Content.Server/Solar/Components/SolarPanelComponent.cs

```diff
@@ -16,67 +15,33 @@ namespace Content.Server.Solar.Components
     ///     It generates power from the sun based on coverage.
     /// </summary>
     [RegisterComponent]
-    public class SolarPanelComponent : Component, IBreakAct
+    public class SolarPanelComponent : Component
     {
         public override string Name => "SolarPanel";
 
         /// <summary>
         /// Maximum supply output by this panel (coverage = 1)
         /// </summary>
-        [DataField("maxsupply")]
+        [DataField("maxSupply")]
         private int _maxSupply = 1500;
+
         [ViewVariables(VVAccess.ReadWrite)]
         public int MaxSupply
         {
             get => _maxSupply;
             set {
                 _maxSupply = value;
-                UpdateSupply();
+                // This should almost never be called, shush.
+                EntitySystem.Get<PowerSolarSystem>().UpdateSupply(OwnerUid, this);
             }
         }
```

Comment:
just make it a friend of the system & get rid of the property imo

Reply:
I knew someone was going to comment on this...
Look, if I do that, it can't be VV'd